File: CS-Examples/23_Worksheets/RemovePageBreak.vb
Imports Spire.Xls

Namespace RemovePageBreak

	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub btnRun_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnRun.Click
            ' Create a new instance of the Workbook class.
            Dim workbook As New Workbook()

            ' Load a workbook from a specified file path.
            workbook.LoadFromFile("..\..\..\..\..\..\Data\PageBreak.xlsx")

            ' Get the first worksheet in the workbook.
            Dim sheet As Worksheet = workbook.Worksheets(0)

            ' Clear all vertical page breaks in the worksheet.
            sheet.VPageBreaks.Clear()

            ' Remove the horizontal page break at index 0 in the worksheet.
            sheet.HPageBreaks.RemoveAt(0)

            ' Set the worksheet view mode to Preview.
            sheet.ViewMode = ViewMode.Preview

            ' Specify the output file name for saving the modified workbook.
            Dim output As String = "RemovePageBreak.xlsx"

            ' Save the workbook to the specified file path using Excel 2013 format.
            workbook.SaveToFile(output, ExcelVersion.Version2013)

            ' Release the resources used by the workbook
            workbook.Dispose()

            'Launch the Excel file
            ExcelDocViewer(output)
		End Sub
		Private Sub ExcelDocViewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub

		Private Sub btnClose_Click(ByVal sender As Object, ByVal e As EventArgs) Handles btnClose.Click
			Close()
		End Sub

	End Class
End Namespace

File: CS-Examples/23_Worksheets/RemovePageBreak.cs
using System;
using System.Windows.Forms;
using Spire.Xls;

namespace RemovePageBreak
{

	public partial class Form1 : Form
	{
        public Form1()
        {
            InitializeComponent();
        }

		private void btnRun_Click(object sender, EventArgs e)
		{
            // Create a workbook
			Workbook workbook = new Workbook();

            // Load the document from disk
            workbook.LoadFromFile(@"..\..\..\..\..\..\Data\PageBreak.xlsx");
             
            // Get the first worksheet from the workbook
            Worksheet sheet = workbook.Worksheets[0];

            // Clear all the vertical page breaks
            sheet.VPageBreaks.Clear();

            // Remove the firt horizontal Page Break
            sheet.HPageBreaks.RemoveAt(0);

            // Set the ViewMode as Preview to see how the page breaks work
            sheet.ViewMode = ViewMode.Preview;

            // Save the document
            string output = "RemovePageBreak.xlsx";
			workbook.SaveToFile(output, ExcelVersion.Version2013);

            // Dispose of the workbook object to release resources
            workbook.Dispose();
            // Launch the Excel file
            ExcelDocViewer(output);
		}
        private void ExcelDocViewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }

        private void btnClose_Click(object sender, EventArgs e)
        {
            Close();
        }

	}
}
